bucket/notepad2-mod.json

```diff
@@ -19,6 +19,9 @@
             "Notepad2-mod"
         ]
     ],
+    "persist": [
```

Comment:
Does this file exists in downloaded archive??
```suggestion
    "persist": "Notepad2.ini",
```

Reply:
yes
![image](https://user-images.githubusercontent.com/7091451/60732845-b5b86600-9f7d-11e9-82ea-174af1fd833c.png)